### PR TITLE
feat(internal/godocfx): mark status of deprecated items

### PR DESCRIPTION
**Pull request by tbpg**
A separate PR will update the template to use this new field.

A future PR here may include a `status` for the overall package and in the TOC.

### CODE REVIEW

**Review by noahdietz — APPROVED** [review]

File: internal/godocfx/parse.go [hunk 1]
```diff
@@ -310,6 +319,16 @@ func parse(glob string, workingDir string, optionalExtraFiles []string, filter [
 	}, nil
 }
 
+// getStatus returns a possibly empty status string for the given
+// docs.
+func getStatus(doc string) string {
+	deprecated := "\nDeprecated:"
```
Comment (dandhlee) [inline internal/godocfx/parse.go:325]: Would it always work with a newline prefix? Is it possible that it's embedded within a line and only has a whitespace prefix instead?

**Review by dandhlee — APPROVED** [review]
LGTM, one minor concern below.

File: internal/godocfx/parse.go (hunk 1 above)
Comment (tbpg) [reply internal/godocfx/parse.go:325]: We may even be able to strengthen this to be two newlines before, but I think this is fine. See https://github.com/golang/go/wiki/Deprecated.

**Review by tbpg — COMMENTED** [review]
Thank you!

**Review by broady — APPROVED** [review]

**Pull request merged**